spec/student_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../student'
+
+describe Student do
+  before(:each) do
+    @student = Student.new(12, 'Agustina', false, '78')
+  end
+
+  it 'Should return the values provided to the constructor class' do
+    expect(@student.age).to eql 12
+    expect(@student.name).to eql 'Agustina'
+    expect(@student.parent_permission).to be false
+    expect(@student.classroom).to ql '78'
```

Comment:
- [ ] You have a failed test due to a misspelling. You spelled` eql` as` ql`, causing your test to fail. Kindly fix that 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/196003466-e3ee97c6-024c-47c8-9b3f-bfaebce27b4a.png)

Reply:
Thank you @vickymarz is done ✅